frontend/src/redux/reducers/ui.tsx

```diff
@@ -206,6 +211,47 @@ function reducer(state = _.cloneDeep(INITIAL_STATE), action: Action) {
       newFilters.branding.logo = component;
       break;
     }
+    case UI_SET_NOTIFICATIONS: {
+      const notifications = action.notifications;
+      /* There are two ways user can send notifications either a complete set of array or a single notification
+         when handling the array notifications we want to only have unique set of notifications pushed into the UI notifications config
+      */
+      if (Array.isArray(notifications)) {
```

Comment:
Should this support the following?
- single notifications adding to the existing notification
- multiple notifications overwriting the existing ones

I see that Notifications/index@Notifications Event.useList always overwrites them, so the single adding notification function will not work. 

So, something needs to be fixed here.

Reply:
Ok, I got confused... for the events adding it looks at the existing event in the store are also left there. (I wonder if this notification logic should be in one place?)

Note: There's a updateUINotification which takes a single Notification for updating.

Reply:
fixed